agent/acs/handler/acs_handler.go

```diff
@@ -200,6 +200,7 @@ func (acsSession *session) Start() error {
 			if shouldReconnectWithoutBackoff(acsError) {
 				// If ACS closed the connection, there's no need to backoff,
 				// reconnect immediately
+				seelog.Info("ACS Websocket connection closed for a valid reason")
```

Comment:
Is it worth logging the error?

Reply:
I opted to remove it from Info level logging since it seemed to lead to confusion more than anything. However, it is still logged at Debug in the wsclient code [here](https://github.com/aws/amazon-ecs-agent/pull/964/files#diff-944db589e66167aa4670b399248d81d0R295)